content/index.html

```diff
@@ -43,25 +42,23 @@ <h1 class="masthead-left__title no-link">
 <main>
   <div class="lessons container container--lp">
     <p class="lessons__learned">
-      Google's web platform team has spent over a decade learning about user
-      needs. Now we want to make it as easy as possible for you to master the
-      defining standards of web development today.
+      As the web advances, users' expectations get higher. With web.dev's guidance, you can give your users the best possible experience, wherever they are.
```

Comment:
This is overflowing onto three lines, and looks a bit funky. Could we move a couple of words to get this to fit on two lines? Maybe 'As the web advances, users' expectations grow'? Or 'With web.dev's guidance, give your users the best possible experience, wherever they are.' I kind of like the first option, and I think it will make it work on two lines.

Reply:
Changed 1st sentence's verb as you recommended; in 2nd sentence, removed "possible." Does it fit on 2 lines now?